config/ssh.yaml

```diff
@@ -0,0 +1,18 @@
+# ssh configuration
+
+# cluster_id: <id of the cluster to connect to, reccommended to specify with --id command line parameter>
+
+# username: <name of the user account to ssh into>
+username: spark
+
+# job_ui_port: <local port where the job ui is forwarded to>
+job_ui_port: 8080
```

Comment:
i think we should just stick with the standard jobui=4040, webui=8080 and jupyter=8888

those would be the ports that users expect.